project/Settings.scala

```diff
@@ -0,0 +1,39 @@
+import sbt._
+import org.scalajs.sbtplugin.ScalaJSPlugin.autoImport._
+
+/**
+ * Application settings. Configure the build for your application here.
+ * You normally don't have to touch the actual build definition after this.
+ */
+object Settings {
+  /** The name of your application */
+  val name = "anduin-fsm"
```

Comment:
It should be inferred automatically from name of variables which hold project definitions in `build.sbt`.